tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_profile_info.py

```diff
@@ -144,36 +204,242 @@ def get_volume_profile_info(volName):
 def get_metrics():
     global CONFIG
     global CLUSTER_TOPOLOGY
+    global READ_WRITE_OPS
+    global LOCK_OPS
+    global INODE_OPS
+    global ENTRY_OPS
     ret_val = {}
     volumes = CLUSTER_TOPOLOGY.get('volumes', [])
     for volume in volumes:
         volName = volume['name']
         vol_iops = get_volume_profile_info(volName)
         if not vol_iops:
             return ret_val
+        read_write_hits = 0
+        inode_hits = 0
+        entry_hits = 0
+        lock_hits = 0
         for brick_det in vol_iops.get('bricks', {}):
             brickName = brick_det.get('brick', '')
-            t_name = "clusters.%s.volumes.%s.nodes.%s.bricks.%s.iops." \
-                "gauge-read"
-            ret_val[
-                t_name % (
-                    CONFIG['integration_id'],
-                    volName,
-                    brickName.split(':')[0].replace('.', '_'),
-                    brickName.split(':')[1].replace('/', '|')
-                )
-            ] = brick_det.get('cumulativeStats').get('totalRead')
-            t_name = "clusters.%s.volumes.%s.nodes.%s.bricks.%s.iops." \
-                "gauge-write"
-            ret_val[
-                t_name % (
-                    CONFIG['integration_id'],
-                    volName,
-                    brickName.split(':')[0].replace('.', '_'),
-                    brickName.split(':')[1].replace('/', '|')
+            brick_host = brick_det.get('hostname', '')
+            if (
+                brick_host == CONFIG['peer_name'] or
+                brick_host == socket.gethostbyname(
+                    CONFIG['peer_name']
                 )
-            ] = brick_det.get('cumulativeStats').get('totalWrite')
-            gevent.sleep(0.1)
+            ):
+                t_name = "clusters.%s.volumes.%s.nodes.%s.bricks.%s.iops." \
+                    "gauge-read"
+                ret_val[
+                    t_name % (
+                        CONFIG['integration_id'],
+                        volName,
+                        CONFIG['peer_name'].replace('.', '_'),
+                        brickName.split(':')[1].replace('/', '|')
+                    )
+                ] = brick_det.get('intervalStats').get('totalRead')
+                t_name = "clusters.%s.volumes.%s.nodes.%s.bricks.%s.iops." \
+                    "gauge-write"
+                ret_val[
+                    t_name % (
+                        CONFIG['integration_id'],
+                        volName,
+                        CONFIG['peer_name'].replace('.', '_'),
```

Comment:
https://github.com/Tendrl/commons/pull/689/files#diff-059917ac9f73bcf3a89b77a6b4806857L92
provides it to https://github.com/anmolbabu/node_agent/blob/e870504523d9822531c77e2c5bbfc6fba0f16965/tendrl/node_agent/monitoring/collectd/templates/gluster/tendrl_glusterfs_profile_info.jinja#L10
and that will be read by the plugin as in https://github.com/Tendrl/node-agent/pull/578/files/e870504523d9822531c77e2c5bbfc6fba0f16965#diff-a0ada298172a6c6cb3782204f212344dL201 and hence will be available here..